### ISSUE TITLE
ARC delay is making revisors to lose done revisions

### ISSUE THREAD

**Issue by paulodamaso** [zerocracy/farm#1164]
It happened four times with me last week:
- `DEV` made changes
- `REV` made revision, told `@rultor` to merge
- `ARC` took too long to merge, changes made by `DEV` gets conflicting with other changes OR `@rultor` have some problem merging;
- `DEV` correct conflicts
- `REV` made review again and says it's okay to  merge
- `ARC` took too long to merge again
- `0crat` removes `REV` from task

This is making reviewers work more than their allotted time and sometimes for free: `REV` cannot put  task in `hold` status (I don't know why), and its revision could potentially be messed up by `ARC` actions.

I see a few alternatives for resolving or mitigating this:
- allowing `REV` jobs to be put on hold, preventing `0crat` from removing tasks when `ARC` is taking too long to merge;
- create a status to label for these issues that were already reviewed, approved and are waiting to be merged, and don't let `0crat` touch these issues while they are waiting to be merged;
- take approved reviews off `0crat` sight

These are the PR where thisk problem occured:
- https://github.com/yegor256/jpeek/pull/245
- https://github.com/yegor256/jpeek/pull/247
- https://github.com/yegor256/jpeek/pull/256

**Comment by 0crat** [comment]
@g4s8[/z](https://www.0crat.com/u/g4s8) please, pay attention to this issue

**Comment by g4s8** [comment]
@paulodamaso you're right, it's a known issue, I think it will be improved with #1028, we are going to add negative points to ARC if PR was not closed fast

**Comment by paulodamaso** [comment]
@g4s8 I agree that we must punish `ARC` on delay, like #1028, but it will not solve the problem at all. The problem is that `0crat` is taking away an job already done. So, if `ARC`does not care about punishment, or just have something more important to attend to and cannot make the merge at the time `REV` work will be wasted anyway.

I think we should refine `0crat` rules by not allowing `0crat` remove a job that has already been approved from a revisor; `REV` had already made his/her job here, so it's `ARC` turn now.

In fact, maybe the countdown for `ARC` punishment should just count after approval; this way we prevent `REV` being affected by `ARC` delay and the same too, a slow `REV` would not make `ARC` pay for it, how about it?
